### PR TITLE
LP-2670 - Fix Webinar issues

### PR DESCRIPTION
**Pull request by MunamMubashir-arbisoft**
[LP-2670](https://philanthropyu.atlassian.net/browse/LP-2670)

- Add length restriction on webinar title field
- Remove virtual field and make meeting_url field required
- Fix the search functionality in webinar registrations
- Add User model as read-only in ADG admin to fix user related field lookup
- Add unit tests for all the added code

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1767/workflows/workflows_id/jobs/1767/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/webinars/admin.py (100%)
openedx/adg/lms/webinars/models.py (100%)
openedx/adg/lms/webinars/tests/factories.py (100%)
openedx/adg/lms/webinars/tests/test_admin.py (100%)
-------------
Total:   35 lines
Missing: 0 lines
Coverage: 100%
-------------
```

File: openedx/adg/lms/webinars/tests/test_admin.py [hunk 1]
```diff
@@ -162,3 +174,38 @@ def test_save_related_send_emails(request, webinar_admin_instance, webinar, mock
         mock_send_webinar_emails.assert_called_once_with(
             MandrillClient.WEBINAR_CREATED, webinar, []
         )
+
+
+def test_readonly_user_admin_change_permission(readonly_user_admin):
+    """
+    Test if ReadOnlyUserAdmin does not have any change permissions
```
Comment (harris-irfan) [inline openedx/adg/lms/webinars/tests/test_admin.py:181]: "Test that" sound more appropriate instead of "Test if"

Please update all occurences.

Comment (MunamMubashir-arbisoft) [reply openedx/adg/lms/webinars/tests/test_admin.py]: Done!

**Review by harris-irfan — APPROVED** [review]
LGTM 👍

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1769/workflows/workflows_id/jobs/1769/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/webinars/admin.py (100%)
openedx/adg/lms/webinars/models.py (100%)
openedx/adg/lms/webinars/tests/factories.py (100%)
openedx/adg/lms/webinars/tests/test_admin.py (100%)
-------------
Total:   35 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by ahmed-faraz46 — APPROVED** [review]
Good work Munam! 👏

**Review by javeriaejaz-arbisoft — APPROVED** [review]
LGTM :+1:

**Pull request merged**